src/auth/user-record.ts

```diff
@@ -148,7 +148,7 @@ export enum MultiFactorId {
  */
 export abstract class MultiFactorInfo {
   public readonly uid: string;
-  public readonly displayName: string | null;
+  public readonly displayName: string;
```

Comment:
Hmm, actually this is nullable. Check below: 
`utils.addReadonlyGetter(this, 'displayName', response.displayName || null);`

If we can't fix in the type definitions, then we should modify below to not be nullable.

Reply:
I think changing typings from optional to nullable can be a breaking change. I think fixing the implementation to match the public API declaration is the right move here.